core/quantity/src/Formatter/Format.ts

```diff
@@ -161,6 +170,22 @@ export class BaseFormat {
         throw new QuantityError(QuantityStatus.InvalidJson, `The Format ${this.name} is 'Ratio' type therefore the attribute 'ratioType' is required.`);
 
       this._ratioType = parseRatioType(formatProps.ratioType, this.name);
+
+      if (undefined !== formatProps.ratioSeparator) {
```

Comment:
nit: yoda conditions here, but below non-yoda/traditional conditions - I would choose one and stick with it.

Reply:
It's not the biggest concern, thanks.